lib/logger.js

```diff
@@ -142,12 +143,18 @@ function Logger(key, options) {
 
     const useHttps = configs.AGENT_PROTOCOL === 'https';
 
+    const headers = (typeof window === 'object') ? {
+        ...configs.DEFAULT_REQUEST_HEADER
+        , source: `${options.UserAgent}/${pkg.version} (${browser.name}/${browser.version})`
```

Comment:
considering `options.userAgent` is already containing `pkg.version` in itself, change this line to
```
,  source: `${options.userAgent || configs.DEFAULT_USER_AGENT} (${browser.name}/${browser.version})`
```